### PR TITLE
Align `"encoding"` option in node fs with node

### PR DESCRIPTION
**Pull request by Jarred-Sumner**
### What does this PR do?

Align `"encoding"` option in node fs with node

### How did you verify your code works?

Copied a test from node

### CODE REVIEW

**Comment by robobun** [comment]
@Jarred-Sumner, your commit 0d6d6c6d4d41238cef193f51191bb0c738119491 is building: [`#5548`](https://buildkite.com/bun/bun/builds/5548)

File: src/bun.js/bindings/bindings.zig [hunk 1]
```diff
@@ -4012,6 +4012,32 @@ pub const JSValue = enum(JSValueReprInt) {
         return JSC__JSValue__getDirectIndex(this, globalThis, i);
     }
 
+    pub fn isFalsey(this: JSValue) bool {
```
Comment (paperclover) [inline src/bun.js/bindings/bindings.zig:4015]: use toBooleanSlow. when bindings like this are rewritten, care should be taken to match exactly how JSC or the spec does them. this implementation does not consider BigInt

**Review by paperclover — CHANGES_REQUESTED** [review]
one insane nit but we shouldnt add these two new functions

otherwise fine

File: src/bun.js/bindings/bindings.zig (hunk 1 above)
Comment (paperclover) [reply src/bun.js/bindings/bindings.zig:4015]: image

image

**Review by paperclover — APPROVED** [review]

**Pull request merged**